setup.py

```diff
@@ -17,7 +17,7 @@
     long_description_content_type="text/markdown",
     project_urls=PROJECT_URLS,
     author_email="yfhe@ucdavis.edu",
-    version="0.0.1.2",
+    version="0.0.1.3",
     packages=find_packages(),
-    python_requires=">=3.10",
+    python_requires=">=3.8",
```

Comment:
Is there a reason for downgrading? I don't want to have two versions of python3 in carcinoma unless we absolutely have to.

Reply:
OSS-Fuzz uses python3.8 in their docker, so this package needs to be compatible with py3.8. 
Downgrading will not require two versions of python3 on our machines since the requirement is `>=` and our version is 3.10.